scripts/update-registry/update.js

```diff
@@ -82,6 +82,10 @@ function update(registry, describeResult) {
       registry.strictDirectoryNames[type.directoryName] = typeId;
     }
   }
+  for (const typeId of Object.keys(registry.types)) {
+    !registry.types[typeId].inFolder && delete registry.types[typeId].inFolder
+    !registry.types[typeId].strictDirectoryName && delete registry.types[typeId].strictDirectoryName
+  }
```

Comment:
What problems does it cause to leave these fields set to false in the registry?  What problems are we solving by deleting them?  Is this just a file size issue or does the update script create less diffs this way?

Reply:
it is a cosmetic change to create less diffs as both properties are not set for all types in the current file. This helped me to review the registry.json file more easily after the script run.